examples/mongocxx/query_projection.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+
+#include <bsoncxx/builder/stream/document.hpp>
+#include <bsoncxx/json.hpp>
+
+#include <mongocxx/client.hpp>
+#include <mongocxx/options/find.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/uri.hpp>
+
+using bsoncxx::builder::stream::document;
+using bsoncxx::builder::stream::open_document;
+using bsoncxx::builder::stream::close_document;
+using bsoncxx::builder::stream::finalize;
+
+int main(int, char **) {
+  mongocxx::instance inst{};
```

Comment:
My quick experiments led to problems.  Let's do a standalone pool example per CXX-995 and then we can retro fit that.